skypy/power_spectrum/_eisenstein_hu.py

```diff
@@ -10,32 +16,39 @@
 
 
 def transfer_with_wiggles(wavenumber, A_s, n_s, cosmology, kwmap=0.02):
-    """ Eisenstein & Hu fitting formula for the transfer function with
-    baryon acoustic oscillation wiggles.
+    r''' Eisenstein & Hu transfer function with wiggles.
+    This function returns the Eisenstein & Hu fitting formula for the transfer
+    function with baryon acoustic oscillation wiggles. This is described in
+    [1]_ and [2]_.
 
     Parameters
     ----------
-    wavenumber : array_like
-        Array of wavenumbers in units of [Mpc^-1] at which to evaluate the
-        linear matter power spectrum.
+    wavenumber : (nk,) array_like
+        Array of wavenumbers in units of :math:`[Mpc^-1]` at which to evaluate
```

Comment:
This does not render well: <img width="121" alt="image" src="https://user-images.githubusercontent.com/3993688/87303375-fb9df780-c50a-11ea-83d9-1c99aaa9b469.png">

(I also don't like the angular brackets but that's just me.)